src/__tests__/createActions-test.js

```diff
@@ -0,0 +1,212 @@
+import { createActions } from '../';
+import { isFSA } from 'flux-standard-action';
+
+describe('createActions', () => {
+  describe('createActions(actionMap)', () => {
+    it('returns an object with actions for each key', () => {
+      const a = createActions({
+        one: (b, c) => ({ bat: b, cat: c }),
+        two: {
+          payload: (d, e) => ({ dog: d, egg: e }),
+          meta: (/* d, e */) => ({ zed: 1 })
+        },
+        three: {
+          meta: (/* f */) => ({ zed: 2 })
+        }
+      });
+      expect(a).to.be.instanceOf(Object);
+      const a1 = a.one(1, 2);
+      expect(a1).to.eql({ type: 'one', payload: { bat: 1, cat: 2 } });
+      expect(isFSA(a1)).to.be.true;
+      expect(a.one.toString()).to.equal('one');
+      const a2 = a.two(3, 4);
+      expect(a2).to.eql({
+        type: 'two',
+        payload: { dog: 3, egg: 4 },
+        meta: { zed: 1 }
+      });
+      expect(isFSA(a2)).to.be.true;
+      expect(a.two.toString()).to.equal('two');
+      const a3 = a.three('foo');
+      expect(a3).to.eql({
+        type: 'three',
+        payload: 'foo',
+        meta: { zed: 2 }
+      });
+      expect(isFSA(a3)).to.be.true;
+      expect(a.three.toString()).to.equal('three');
```

Comment:
Why is testing `toString()` of the action creators important? As a library user I'd never use this value; I'd just care about the actions produced.